docs/src/main/sphinx/connector/metastores.md

```diff
@@ -432,6 +432,12 @@ properties:
   * - ``iceberg.nessie-catalog.transport.enable-compression``
     - Configure whether compression should be enabled or not for
       requests to the Nessie server. Defaults to ``true``.
+  * - ``iceberg.nessie-catalog.authentication.type``
+    - The authentication type to use.
+      Available values are ``NONE`` or ``BEARER``. Defaults to ``NONE``.
```

Comment:
The property doesn't allow `NONE`. Please update this sentence.

Reply:
done